include/utils.bash

```diff
@@ -19,6 +19,14 @@ gen-password() {
     date +%s | checksum sha1 | head -c 10
 }
 
+host-from-url() {
```

Comment:
Would be nice using go code for utility functions. There are exaples in 'functions.go' and how to register them as bash functions in `deployer.go`. It's lot easier to test these functions as go code ('functions_test.go')

Reply:
Okay, while my Golang knowledge is a big fat zero, I am looking into how to do this. 😃 Will be back soon with updates.

Reply:
@bbihari These two utility functions have been translated into Go. Could you please re-check?